components/Dashboard/candidate/profile/add-education-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import React,{useEffect, useState} from 'react'
+import Modal from '@/components/modal/Modal'
+import { CancelIcon } from '@/assets';
+import { DateInput, Input } from '@/components/Input';
+
+import { Button, OutlineBtn } from '@/components/button';
+import PrimarySelect from './primary-selects';
+interface modalcompProps {
+    open: boolean;
+    setOpen: any;
+    onClick: any;
```

Comment:
`onClick: () =>  void`

add the right typescript types here and to your useStates.